glfw/src/Joystick.zig

```diff
@@ -438,17 +464,21 @@ pub inline fn isGamepad(self: Joystick) bool {
 /// @return The UTF-8 encoded name of the gamepad, or null if the joystick is not present or does
 /// not have a mapping.
 ///
+/// Possible errors include glfw.Error.NotInitialized and glfw.Error.InvalidEnum.
+///
```

Comment:
Ah, just to make sure I understand this addition, we're adding it to keep in sync with GLFW documentation since every other function has these - but the function can't actually return these.

Is `InvalidEnum` as is documented here correct? Should it be `InvalidValue` instead?

Reply:
Yes, that's exactly right; and yes, it is `InvalidEnum`, as I mentioned in https://github.com/glfw/glfw/pull/2007; there's no calls to set `InvalidValue` in any of the top level source or in the possible implementation functions, so that's the set.